### PR TITLE
fix: missing default borderRadius-xl

### PR DESCRIPTION
**Pull request by NamesMT**
Also make quotes consistent

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -95,8 +95,9 @@ export function presetShadcn(options: PresetShadcnOptions = {}): Preset<Theme> {
         },
       },
       borderRadius: {
-        lg: `var(--radius)`,
-        md: `calc(var(--radius) - 2px)`,
+        xl: 'calc(var(--radius) + 4px)',
```
Comment (hyoban) [inline src/index.ts:98]: I didn‘t see the shadcn document with the recommendation to set this line.

Comment (NamesMT) [reply src/index.ts:98]: Hi @hyoban, you can see the declarations here: https://github.com/search?q=repo%3Aradix-vue%2Fshadcn-vue%20calc(var(--radius)%20%2B%204px)&type=code

Comment (hyoban) [reply src/index.ts:98]: https://github.com/search?q=repo%3Aradix-vue%2Fshadcn-vue+path%3A%2F%5Eapps%5C%2Fwww%5C%2Fsrc%5C%2Flib%5C%2Fregistry%5C%2Fdefault%5C%2Fui%5C%2F%2F+rounded-xl&type=code

It seems like `rounded-xl` is not used

Comment (hyoban) [reply src/index.ts:98]: Oh, It looks like the cli will generate this configuration

**Review by hyoban — APPROVED** [review]

**Pull request merged**